data/json/items/corpses/inactive_bots.json

```
@@ -269,6 +269,28 @@
       "skill2": "computer"
     }
   },
+  {
+    "id": "bot_turret_riot",
```

```suggestion
    "id": "bot_talon_m203",
```
Consistent IDs for our UGVs makes it easier to know exactly what they are by ID only.